doc/sphinx/addendum/extraction.rst

```diff
@@ -471,6 +471,12 @@ Additional settings
 
    If this :term:`flag` is set, fully expand Coq types in ML.  See the Coq source code to learn more.
 
+.. opt:: Extraction Output Directory @string
+
+   Sets the directory in which the extracted `.ml` (and `.mli`) files will be
+   written. The default is the current directory which can be found using
+   :cmd:`Pwd`.
```

Comment:
Does it take a relative or absolute path?

Reply:
It can take both. Of course, relative is probably more sensible.